src/DynamoRevit/DynamoRevit.cs

```diff
@@ -481,8 +481,7 @@ internal static Version PreloadAsmFromRevit()
             Version libGversion = findRevitASMVersion(asmLocation);
 
             var dynCorePath = DynamoRevitApp.DynamoCorePath;
-            var libGFolderName = string.Format("libg_{0}_{1}_{2}", libGversion.Major, libGversion.Minor, libGversion.Build);
-            var preloaderLocation = Path.Combine(dynCorePath, libGFolderName);
+            var preloaderLocation = DynamoShapeManager.Utilities.GetLibGPreloaderLocation(libGversion, dynCorePath);
```

Comment:
@QilongTang would you mind adding a comment here to briefly document the fallback behavior? - Or does this method have a summary tag that shows up?

Reply:
The summary does not show up for me in my VS, I can add comments but will be the same with function signature